### PR TITLE
(DOCSP-19709) Changelog

### PR DESCRIPTION
**Pull request by biniona-mongodb**
## Pull Request Info

Add "What's New" page

### Issue JIRA link:

https://jira.mongodb.org/browse/DOCSP-19709

### Snooty build log:

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?jobId=61c0f10ec52f2a460f6a07a6

### Docs staging link (requires sign-in on MongoDB Corp SSO):

https://docs-mongodbcom-staging.corp.mongodb.com/visual-studio-extension/docsworker-xlarge/DOCSP-19709-Changelog/whats-new/

### Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Does it render on staging correctly?
- [x] Are all the links working?
- [x] Are the staging and workerpool job links in the PR description updated?

### If your page documents a concept, does it meet the following criteria?

- [ ] Target the [Jasmin persona](https://drive.google.com/file/d/14FbBOLCVxwSP6M9BK4Nz1Ir9tzxT8_02/view)
- [ ] Target the [Lucas persona](https://drive.google.com/file/d/1J2vqJxo7ldv7OP_obA9Q-avf0o_ju4Lk/view)

Note: This is almost a direct copy and paste of the [Kafka Version 1.0 Changelog](https://github.com/10gen/docs-kafka-connector/blob/v1.0/source/whats-new.txt)

### CODE REVIEW

File: source/whats-new.txt [hunk 1]
```diff
@@ -1 +1,18 @@
...
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+What's New in 1.0
```
Comment (ccho-mongodb) [inline source/whats-new.txt:15]: Could be helpful to have a label here for direct linking for when you have multiple versions.

File: source/whats-new.txt [hunk 2]
```diff
@@ -1 +1,18 @@
...
+-----------------
+
+The initial GA release.
```
Comment (ccho-mongodb) [inline source/whats-new.txt:18]: Please expand "GA" to general availability or omit it. It's not a term that everyone uses nor is familiar with.

**Review by ccho-mongodb — APPROVED** [review]
LGTM, a couple comments.

File: source/whats-new.txt (hunk 2 above)
Comment (biniona-mongodb) [reply source/whats-new.txt:18]: Removed "GA". Now it's just "The initial release."

File: source/whats-new.txt (hunk 1 above)
Comment (biniona-mongodb) [reply source/whats-new.txt:15]: Added a ref!

**Review by mmarcon — APPROVED** [review]

**Pull request merged**